ietf/secr/templates/includes/sessions_request_form.html

```diff
@@ -118,7 +118,7 @@
     <div class="button-group">
         <ul>
             <li><button type="submit" name="submit" value="Save"{% if is_locked %} disabled{% endif %}>Save</button></li>
-            <li><button type="submit" name="submit" value="Cancel">Cancel</button></li>
+            <li><button type="button" onclick="window.location='{% url "ietf.secr.sreq.views.main" %}'">Back</button></li>
```

Comment:
```suggestion
            <li><a type="button" href="{% url 'ietf.secr.sreq.views.main' %}">Back</a></li>
```

Reply:
Or does this template not use bs5?

Reply:
Bootstrap is available, but not used on the overall template.  I could pull it in if desired.

If so, then I'll probably make the same changes to `ietf/secr/templates/sreq/view.html` as well, since I copied it out of there.

Reply:
Oh, I just realized this is wrong anyway - I should have it go back through the history instead of specifying a specific target, because this form is included on two paths, which have different navigation.